hera_cal/_cli_tools.py

```diff
@@ -0,0 +1,351 @@
+"""Useful helper functions and argparsers for scripts."""
```

Comment:
If this is boilerplate code, put in a reference to it's origin / some explanation. If not, then I think it'd be helpful to document all of these functions.